rtdb2/RtDB2.cpp

```diff
@@ -486,3 +481,24 @@ void RtDB2::decompress(std::string &s)
     }
 }
 
+std::vector<int> RtDB2::getAgentIds() const
```

Comment:
Is 'int' the best type, or should we use some kind of PlayerId?

Reply:
I agree, but I do not consider that part of this change/pull request. Currently an agent id is an int in RTDB.